.github/workflows/pythonpackage.yml

```diff
@@ -29,6 +29,7 @@ jobs:
     - run: which python
     - run: conda config --set always_yes yes --set changeps1 no
     - run: conda install -c conda-forge lammps mpi4py openmpi pytest numpy
+    - run: pip install distributed dask-jobqueue
```

Comment:
Is there a specific reason why you install the packages via pip rather than conda?